publications-2016.bib

```diff
@@ -2224,4 +2224,31 @@ @Article{rudraraju16:_mechan
   url =    {10.1038/npjcompumats.2016.12}
 }
 
+@misc{sticko2016higher,
+    title={Higher Order Cut Finite Elements for the Wave Equation},
+    author={Simon Sticko and Gunilla Kreiss},
+    year={2016},
+    eprint={1608.03107},
+    archivePrefix={arXiv},
+    primaryClass={math.NA}
+}
```

Comment:
I think that we can update this since it has been published in JSC:
https://link.springer.com/article/10.1007/s10915-019-01004-2

Reply:
I'm torn what to do in such cases. I have a paper for which the arXiv preprint is far more detailed than the final published version. It's actually the one below, and so I thought it makes sense to list both the preprint and the final version. In the current case, it took the authors 3 years to get it published, so I'm tempted to leave the preprint in as well.

Reply:
I'm updating the other one, though.

Reply:
Thats a reasonable argument for leaving it with the arxiv link.